rendered/manifests/agent-only/configmap-agent.yaml

```diff
@@ -188,44 +188,4 @@ data:
       - k8s_observer
       - memory_ballast
       - zpages
-      pipelines:
-        metrics:
-          exporters:
-          - signalfx
-          processors:
-          - memory_limiter
-          - batch
-          - resource
-          - resourcedetection
-          receivers:
-          - hostmetrics
-          - kubeletstats
-          - otlp
-          - receiver_creator
-          - signalfx
-        metrics/agent:
-          exporters:
-          - signalfx
-          processors:
-          - memory_limiter
-          - batch
-          - resource
-          - resource/add_agent_k8s
-          - resourcedetection
-          receivers:
-          - prometheus/agent
-        traces:
-          exporters:
-          - sapm
-          - signalfx
-          processors:
-          - memory_limiter
-          - k8sattributes
-          - batch
-          - resource
-          - resourcedetection
-          receivers:
-          - otlp
-          - jaeger
-          - smartagent/signalfx-forwarder
-          - zipkin
+      pipelines: null
```

Comment:
Seems like templating is broken. This is invalid output

Reply:
Thanks. Updated the PR